java/src/main/java/com/google/cloud/dataproc/templates/databases/SpannerToGCS.java

```diff
@@ -89,4 +90,29 @@ public void runTemplate() {
 
     spark.stop();
   }
+
+ void validateInput() {
+	   if (StringUtils.isAllBlank(projectId)
```

Comment:
We need to put more comprehensive checks using jakarta validator framework.
Refer to JdbcToGCSConfig for examples.

Reply:
@jitendra390 these checks for SpannerToGCS have been added as part of https://github.com/GoogleCloudPlatform/dataproc-templates/pull/317. Please refer the latest code from main branch.